authz/providers/azure/rbac/rbac.go

```diff
@@ -76,10 +76,22 @@ type AccessInfo struct {
 }
 
 var (
-	checkAccessFailed = promauto.NewCounter(prometheus.CounterOpts{
+	checkAccessThrottled = promauto.NewCounter(prometheus.CounterOpts{
 		Name: "guard_azure_checkaccess_throttling_failure_total",
 		Help: "Azure checkaccess call throttled.",
 	})
+	noOfCheckAccessCalls = promauto.NewCounter(prometheus.CounterOpts{
```

Comment:
should rename to checkAccessTotal or checkAccessCount?

Reply:
done